lib/__tests__/harvest.test.ts

```diff
@@ -230,7 +230,8 @@ describe("Integration tests", () => {
       page.on("response", async interceptedRequest => {
         const request = interceptedRequest.request();
 
-        if (request.url().includes(".algolia.net")) {
+        const url = request.url();
+        if (url.includes(".algolia.net") || url.includes(".algolianet.com")) {
```

Comment:
Not sure if that matters but as `includes` isn't supported in IE, we could use something like
```suggestion
        if ([".algolia.net", ".algolianet.com"].indexOf(url) >= 0) {
```

Reply:
Nevermind 🤔